docker/env/hydra.sh

```diff
@@ -247,7 +247,9 @@ function run_in_docker () {
         -v "${SCT_DIR}:${SCT_DIR}" \
         -v /tmp:/tmp \
         -v /var/tmp:/var/tmp \
-        -v "${HOME_DIR}:${HOME_DIR}" \
+        -v "${HOME_DIR}/.aws:${HOME_DIR}/.aws" \
+        -v "${HOME_DIR}/.ssh:${HOME_DIR}/.ssh" \
+        -v "${HOME_DIR}/sct-results:${HOME_DIR}/sct-results" \
```

Comment:
The precommit failed with the following error:

```
Error saving credentials: mkdir /home/jenkins/.docker: permission denied
```

So, looks like the list of sub-directories mounts is not complete.
At least `${HOME_DIR}/.docker` is missing

Reply:
For K8S may be needed `.helm` and `.kube`...

Reply:
yeah, it would take some time to scope them all out...

Reply:
Oh, it also wants `.cache` dir